assets/src/edit-story/components/canvas/carousel/index.js

```diff
@@ -122,12 +123,7 @@ const buttonDimensions = { width: '24', height: '24' };
 
 const StyledGridViewButton = styled(GridViewButton).attrs(buttonDimensions)``;
 
-const SafeZoneButton = styled(SafeZone).attrs(buttonDimensions)`
-  ${({ active, theme }) =>
-    active &&
-    css`
-      background: ${rgba(theme.colors.bg.white, 0.1)};
-    `}
+const SafeZoneToggle = styled(ToggleButton).attrs(buttonDimensions)`
```

Comment:
Percy is [detecting a slight difference](https://percy.io/google/web-stories-wp/builds/6678009/view/382686082?browser=chrome&mode=base&width=1600) between the button before and now - it's a bit bigger and differently aligned with respect to the button below.

Compare:

| Before | After |
|-|-|
| <img width="112" alt="Screen Shot 2020-08-27 at 16 11 53" src="https://user-images.githubusercontent.com/637548/91490223-1987a780-e880-11ea-84c7-49313f95319e.png"> | <img width="112" alt="Screen Shot 2020-08-27 at 16 11 59" src="https://user-images.githubusercontent.com/637548/91490231-1b516b00-e880-11ea-8418-4799922c191c.png"> |

I don't know which is right (if either), so please check with Figma.

Reply:
![image](https://user-images.githubusercontent.com/2755722/91592277-47c2c100-e92c-11ea-895a-d9d7f19b3552.png)

This should be resolved, the original was okay'd by Sam and the current implementation should now match.